integration/integration.go

```diff
@@ -0,0 +1,30 @@
+//go:build integration
```

Comment:
Let's avoid using random values. I think we can run all test cases in eco-goinfra-intergation-tests namespace.

Reply:
I actually like the random namespaces when running in parallel.  Keeps everything separated nicely and doesn't allow for any cross contamination (if any, potentially) of any of the tests.  We can discuss at next meeting if needed.